prime-router/metadata/HL7/catchall/hl7/datatypes/AUI/AUIExtension.yml

```diff
@@ -2,7 +2,7 @@
 
 url:
   type: STRING
-  valueOf: IN1.14
+  valueOf: $auiExtensionName
```

Comment:
Unsure why this was hardcoded to `IN1.14` previously, but this now better resembles other extension datatypes in that it receives the extension name from the caller.